Build/libHttpClient.141.Android.Java/app/build.gradle

```diff
@@ -2,7 +2,7 @@ apply plugin: 'com.android.model.library'
 
 model {
     android {
-        compileSdkVersion = 19
+        compileSdkVersion = 21
```

Comment:
>compileSdkVersion = 21 [](start = 8, length = 22)

what's the reason for changing the sdk version?

Reply:
it's because Android devices didn't support 64-but before 21.  however, thanks for the comment because it also caused me to go back and see that I still have minSdkVersion set to 19, so I should fix that.